app/scripts/views/home/home.js

```diff
@@ -3,19 +3,75 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 define([
+  'jquery',
+  'fullpage',
   'views/base'
-], function (BaseView) {
+], function ($, fullpage, BaseView) {
   'use strict';
 
   var HomeView = BaseView.extend({
+    events: {
+      'click .skip-intro':'_skipIntro',
+      'click .arrow': '_moveDown'
+    },
+
     initialize: function () {
-      this.$('h2').html('home sweet home');
+      this.mediaNames = ['video', 'story', 'photo', 'article', 'song', 'picture'];
+      this._setRandomColors();
+      this._initializeFullpage();
     },
 
     // the html is delivered in the page by the server. nothing to render.
     render: function () {
       return this;
+    },
+
+    _initializeFullpage: function() {
+      var that = this;
```

Comment:
Sorry, realized I didn't explain what's going on here, and maybe it would be good to add some detail.

We only need to `bind` functions called from a non-object context, like when a click handler is invoked, or from inside a setTimeout. So, if your event handlers all have a `.bind(this)` at the end, you should be able to just use `this` everywhere. And if you're inside a function in which `this` was properly bound, then you can just call `this._otherPrivateFunction` with reckless abandon, because the context is already set correctly.

For example, here's a scroll handler, take a look at how it's wired up and how `this` is used inside the body of the handler: https://github.com/mozilla/chronicle/blob/9dc7beb/app/scripts/views/visits/index.js#L36

If you're attaching listeners using Backbone, as in [this example](https://github.com/mozilla/chronicle/blob/9dc7beb/app/scripts/views/visits/index.js#L36), then `this` is automatically bound to the View instance for you, and life is even easier.

Hope this is helpful, feel free to ask about any of this stuff if you're ever unclear on what the code means...and also feel free tell me to stop trying to be so helpful if it's not, in fact, helping ;-)